packages/core/actions/account-client/getUserOperationGasPrice.ts

```diff
@@ -24,3 +27,21 @@ export const getUserOperationGasPrice = async <
         maxPriorityFeePerGas: BigInt(gasPrice.standard.maxPriorityFeePerGas)
     }
 }
+
+export const pimlicoGetUserOperationGasPrice = async <
```

Comment:
This can be imported from permissionless actions.

Reply:
I think this won't be needed once we remove [this](https://github.com/zerodevapp/sdk/pull/122/files#r1578654333)